src/endpoint/s3/ops/s3_post_bucket_delete.js

```diff
@@ -23,6 +23,8 @@ async function post_bucket_delete(req) {
     }
 
```

Comment:
Notice that on AWS, will we get this error on the non-filtered array, I checked it on an array that contains only 2 different keys, and all the rest are duplicates
so I'm not sure you will not be able to filter first

Reply:
I agree with you.